bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/ClientConfiguration.java

```diff
@@ -797,6 +798,33 @@ public ClientConfiguration setPCBCTimeoutTimerNumTicks(int numTicks) {
         return this;
     }
 
+    /**
+     * Timeout controlling wait on request send in case of unresponsive bookie(s)
+     * (i.e. bookie in long GC etc.)
+     *
+     * @return timeout value
+     *        negative value disables the feature
+     *        0 to allow request to fail immediately
+     *        Default is 500
+     */
+    public long getWaitTimeoutOnBackpressureMillis() {
+        return getLong(WAIT_TIMEOUT_ON_BACKPRESSURE, 500);
```

Comment:
I think we should make this fault value as -1, util we have benchmarks show 500 is in general good for most of applications.